tools/accuracy_checker/accuracy_checker/metrics/metric_profiler/object_detection_metric_profiler.py

```diff
@@ -81,20 +81,19 @@ def generate_json_report(self, identifier, metric_result, metric_name):
 
     def per_box_result(self, identifier, metric_result):
         per_box_results = []
-        for label, per_class_result in enumerate(metric_result):
+        for label, per_class_result in metric_result.items():
```

Comment:
why this behavior changed?
Not all detection metrics return data in dictionary format. some of them return list. Also not always label here is the same like in annotation, so approach with label_id is better